msteams-platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md

```diff
@@ -108,6 +108,17 @@ After you enable Share to Teams button on personal app or tab, you can share the
 
    :::image type="content" source="../../assets/images/share-to-teams/link-shared.PNG" alt-text="share-to-teams-link-shared":::
 
+## Share to meeting from personal app
+
+Teams enables you to share personal apps to meeting stage using the Present now button. This button is available in all personal tab apps providing quicker sharing options to the meeting stage. 
```

Comment:
We should make it clearer that this button is available by default/no developer investment is needed. Otherwise, it looks good!

Reply:
Updated the content to 'Teams enables you to share personal apps to meeting stage using the **Present now** button. This button is available by default in all personal tab apps and doesn't require any developer implementation. It provides quicker sharing options to the meeting stage.'